src/owlwatch/schema/model.py

```diff
@@ -180,10 +180,20 @@ def from_json(cls, ip_json):
         # Get index pattern fields
         fields_json = json.loads(ip_json['value']['fields'])
         for json_field in fields_json:
+            if 'scripted' in json_field and json_field['scripted']:
+                # Don't check Kibana generated scripted fields
+                continue
+            elif 'is_' in json_field['name']:
+                # Don't check is_ fields because they don't exists always
+                # is_gmane only exists in mbox coming from gmane for example
+                continue
+            agg = True
+            if 'aggregatable' in json_field:
+                agg = json_field['aggregatable']
             index_pattern.add_property(pname=json_field['name'],
                                        ptype=json_field['type'],
                                        analyzed=json_field['analyzed'],
-                                       agg=json_field['aggregatable'])
+                                       agg=agg)
```

Comment:
If an Index Pattern doesn't have aggregatable field, it is probably due to an old version of JSON file, not exported from Kibana 5. Thus, we should raise an error or warning and just not adding that property.